docs/rbac.rst

```diff
@@ -0,0 +1,53 @@
+.. _rbac:
+
+RBAC Configuration
+******************
+
+To enhance security, the `edit` `ClusterRoleBinding` has been removed from
+the Kanister Helm Chart. Users now need to create their own `Role` / `RoleBinding`
+in the application's namespace to grant the necessary permissions to
+Kanister's Service Account.
+
+Creating a RoleBinding with edit ClusterRole
+============================================
+
+To allow Kanister to perform backup/restore operations in a specific
+namespace, create a `RoleBinding` that assigns the `edit` `ClusterRole`
+to Kanister's Service Account:
+
+.. code-block:: bash
+
+  kubectl create rolebinding kanister-edit-binding --clusterrole=edit \
+  --serviceaccount=<release-namespace>:<release-name>-kanister-operator \
+  --namespace=<application-namespace>
+
+Creating a Role with Granular Permissions
+=========================================
+
+If you prefer to fine-tune the access, you can create a `Role` with
+specific permissions and bind it to Kanister's Service Account:
```

Comment:
I think we should mention that if your blueprint doesn't require access to all the resources that are mentioned in `edit` cluster role, you can create the role with just the resources/verbs that the blueprint requires access to.

Reply:
Updated